code/modules/research/designs/spacepod/weapons.dm

```diff
@@ -28,3 +28,23 @@
 	category = "Pod_Weaponry"
 	materials = list(MAT_IRON = 15000)
 	locked = 1
+
+/datum/design/pod_lock
+	name = "Spacepod Equipment (Toggle Lock)"
+	desc = "Allows for the construction of a spacepod mounted locking system."
+	id = "pod_lock"
+	build_type = PODFAB
+	build_path = /obj/item/device/spacepod_equipment/locking/lock
+	req_tech = list(Tc_MATERIALS = 2, Tc_BLUESPACE = 2)
```

Comment:
Weird tech requirement but not enough to be controversial. Data or engineering might seem more natural?